src/Tyler.jl

```diff
@@ -200,16 +198,16 @@ function create_tile_plot!(tyler::Map, tile::Tile, image::TileImage)
         mplot = create_tileplot!(tyler.axis, image)
     else
         mplot = pop!(tyler.free_tiles)
-        mplot.visible = true
         mplot.color[] = image
+        mplot.visible = true
     end
     tyler.plots[tile] = mplot
     place_tile!(tile, mplot, tyler.coordinate_system)
 end
 
 function fetch_tile(tyler::Map, tile::Tile)
     return get!(tyler.fetched_tiles, tile) do
-        url = geturl(tyler.provider, tile.x, tile.y, tile.z)
+        url = TileProviders.geturl(tyler.provider, tile.x, tile.y, tile.z)
         result = HTTP.get(url; retry=false, readtimeout=4, connect_timeout=4)
         return ImageMagick.readblob(result.body)
```

Comment:
I recently tried to download tiles from all providers, but for some this `get(; retry)` thing was always on and failing. Q. what should be the return value of this function if the download fails? a noise image? a black one? a transparent one? or none of the above 😄 ? I don't know what this `ImageMagick.readblob(result.body)` is doing it `result` is a failed attempt.

Reply:
Yeah I guess it errors? I had actually hanged that to `true` and shortened the timeouts but decided that's for a different PR, and I also don't really understand the tradeoffs.

Reply:
I see. Ok